ai_agents/personal_finance_agent/app.py

```
@@ -146,6 +170,17 @@ def main() -> None:
                         "`deepseek-v4-flash` available."
                     )
                 st.markdown(answer)
+                chat_export = "\n\n".join(
+                    f"{m['role'].upper()}: {m['content']}"
+                    for m in st.session_state.messages[:-1] + [{"role": "assistant", "content": answer}]
```

Ruff (0.15.15)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)